mrt/models.py

```diff
@@ -400,6 +400,26 @@ def photo(self):
             return photo.value if photo else None
         return None
 
+    @property
+    def address(self):
+        field = self.meeting.address_field
+        if field:
+            address = (
+                field.custom_field_values
+                .filter_by(participant_id=self.id).first())
+            return address.value if address else None
+        return None
+
+    @property
+    def telephone(self):
+        field = self.meeting.telephone_field
+        if field:
+            telephone = (
+                field.custom_field_values
+                .filter_by(participant_id=self.id).first())
+            return telephone.value if telephone else None
+        return None
```

Comment:
These three properties are very similar. Why not write a function that receives the name of the field as a parameter and then call this function inside the properties?